installer/scripts/npm_startup.sh

```diff
@@ -38,6 +38,7 @@ fi
 
 # Create script file
 su - omsagent -c "echo $'chmod 755 \$1\nsetcap cap_net_raw=ep \$1' > $scriptPath"
+chown root:root $scriptPath
```

Comment:
Looks like we have elevated privileges when this script is executed (since we are directly modifying `/etc/sudoers.d/omsagent` which has `root:root` ownership). If this is true we can directly merge the above 2 lines to be:

```
echo $'chmod 755 \$1\nsetcap cap_net_raw=ep \$1' > $scriptPath
```

which should create `NPMAgentBinaryCap.sh` with user and group as `root`. But I'm not sure if my assumption about elevated privileges is correct.

Reply:
Hi, if you are confident about this change can you merge it, we are about to start OMS release soon